src/components/ProfileSwitcher.js

```diff
@@ -0,0 +1,43 @@
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import Dropdown from './Dropdown';
+
+import { setActiveProfile } from '../store/actions';
+
+import styles from './ProfileSwitcher.module.css';
+
+function ProfileSwitcher() {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const activeProfile = useSelector((state) => state.activeProfile);
+  const profiles = useMemo(
+    () => [
+      {
+        value: 'ch',
+        text: t('profiles.bund'),
+      },
+      {
+        value: 'be',
+        text: t('profiles.bern'),
+      },
+      {
+        value: 'lu',
+        text: t('profiles.luzern'),
+      },
+    ],
+    [t],
+  );
+
+  return (
+    <Dropdown
+      value={activeProfile}
+      className={styles['profile-switcher']}
+      options={profiles}
+      onChange={(e, { value }) => dispatch(setActiveProfile(value))}
+      data-cypress="profileSwitcher"
```

Comment:
```suggestion
      data-cypress="profileSwitcher"
      search={false}
```
Maybe we should disable search because it will always be a rather limited list of options.

Reply:
ok